src/ASM/ASMbase.C

```diff
@@ -235,6 +246,15 @@ int ASMbase::getNodeID (size_t inod, bool) const
 }
 
 
+char ASMbase::getLMType(size_t n) const
+{
+  if (n >= myLMs.first && n <= myLMs.second)
+    return myLMTypes[n-myLMs.first];
+
+  return 'L';
+}
```

Comment:
Hvis n er utenfor myLMs rangen er den vel hverken 'L' eller 'G' så da er det mer konsistemt å returnere noe annet, f.eks. 0 eller false. Og kan bruke isLM for selv sjekken, dvs.

  return this->isLMn(n) ? myLMTypes[n-myLMs.first] : 0;